nuspec/nuget/gazorator.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Gazorator</id>
+    <version>0.0.0</version>
+    <authors>mholo65</authors>
+    <owners>mholo65</owners>
+    <description>Enables Razor outside ASP.NET</description>
+    <summary>Enables Razor outside ASP.NET</summary>
+    <licenseUrl>https://github.com/mholo65/gazorator/blob/develop/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/mholo65/gazorator</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <copyright>2018 Martin Björkström</copyright>
+    <tags>razor, gazorator</tags>
+    <dependencies>
+      <dependency id="Microsoft.AspNetCore.Razor.Language" version="2.1.0-preview2-final" />
+      <dependency id="Microsoft.CodeAnalysis.CSharp.Scripting" version="2.7.0" />
+      <dependency id="System.Text.Encodings.Web" version="4.4.0" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="netcoreapp2.0\Gazorator.dll" target="lib/netcoreapp2.0" />
+    <file src="netcoreapp2.0\Gazorator.pdb" target="lib/netcoreapp2.0" />
+  </files>
+</package>
```

Comment:
I'd prefer skipping `nuspec` and having `dotnet pack` generate the `nupkg`. It's much easier to handle dependencies this way. This can be enabled by setting `shouldRunDotNetCorePack: true` in `BuildParameters` (`setup.cake`)